inst/tests/tests.Rraw

```diff
@@ -10977,7 +10977,7 @@ test(1732.7, fwrite(DT, quote='auto'), output='A,B\n,5\nNA,7\n"",0\nmonty,')
 test(1732.8, fwrite(DT, quote='auto', na="NA"), output='A,B\nNA,5\n"NA",7\n"",0\n"monty",NA')
 
 # dec=","
-test(1733.1, fwrite(data.table(pi),dec=","), error=base_messages$stopifnot("dec != sep"))
+test(1733.1, fwrite(data.table(pi),dec=",", file=nullfile()), NULL)
```

Comment:
Just delete this test instead (it's no longer useful); leave a comment in its stead